components/admin/modules/SideBar.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import _ from 'lodash'
+import { Module } from '../../../graphql/index'
+import styles from '../../../scss/sideBar.module.scss'
+import { XIcon, ChevronRightIcon } from '@primer/octicons-react'
+
+type SideBarProps = {
+  modules: Module[] | []
+  selectedModule: number
+}
+export const SideBar: React.FC<SideBarProps> = ({
+  modules,
+  selectedModule
+}) => {
+  const moduleListData = [_.cloneDeep(modules) || []]
+  const lessonList = moduleListData[0].map((obj: any, arrIndex: number) => (
+    <>
+      <a
```

Comment:
Anchor element is used to create a hyperlink. So I think if the `href` prop is passed which means it won't redirect the user to any other page. Maybe it should be turned into a `button`.